portal-web/test/functional/com/liferay/portalweb/tests/MyTestFile/MyTestFile.testcase

```diff
@@ -16,16 +16,14 @@ userLoginFullName = "Test Test");
 Control Panel"
 @priority = "5"
 test ValidateConfigurationStringAtControlPanel {
-task("will open the application menu"){
-        Click(locator1 = "//button[contains(@data-qa-id, 'applicationsMenu')]");
+task("accessing the option menu"){
+        MyTestFile.clickMenu();
 }
-task("access the control panel option"){
-        Click(locator1 = "//*[text()='Control Panel']");
+task("click on control panel option"){
+        MyTestFile.clickControPanel();
 }
-task("checking if the configuration option is in the control panel section"){
-        AssertVisible(locator1 = "//h2[text()='Configuration']");
+task("check the configuration section is present"){
+        MyTestFile.assertConfiguration();
 }
-
-
 }
```

Comment:
This commit(69ae268ab7ab4aae897d049a75c730720fd670cc) does not apply to this ticket, can you please remove and resend? Thanks.